.config/molecule/config_local.yml

```diff
@@ -23,7 +23,7 @@ provisioner:
   inventory:
     group_vars:
       all:
-        cifmw_discover_latest_image_qcow_prefix: "CentOS-Stream-GenericCloud-9-20240401"
+        cifmw_discover_latest_image_qcow_prefix: "CentOS-Stream-GenericCloud-9-20240506"
```

Comment:
I think we can remove that altogether, the upstream image is fixed afaik.

Care to comment out the `inventory` block so that we know how to override that kind of parameter in the future?

Reply:
done, but I'm afraid there will be other unrelated molecule failures:
- https://review.rdoproject.org/zuul/build/8082f1cd117d41df82be84d55ed04795
- https://review.rdoproject.org/zuul/build/cb761caa529d4bf898c03134d5e11903

Reply:
`kustomize_deploy` will be fixed in #1641
`sushy_emulator`.... maybe better check with @lewisdenny

Reply:
sushy_emulator molecule test is slightly flaky atm but completely reworked in https://github.com/openstack-k8s-operators/ci-framework/pull/1560